README.md

```diff
@@ -14,6 +14,31 @@ Run `npm test`
 to the following object (`{memory: true}`) being passed in as a parameter for new 
 sqliteDatabase() located in `searchDao.js`, AND all `Spec js` files in the test directory.
 
+## Documentation
+
+When making a GET call, you can add query parameters. These are all of the parameters you 
+can add.
+
+```
+?class=1&seat=A,B&departureDateAfter=2038-01-11&
+departureDateBefore=2038-01-12&
+arrivalDateAfter=2038-01-16&
+arrivalDateBefore=2038-01-21&
+departure_location=TWS,DSA&
+destination_location=FSE,CAR
+```
+
+All of the query parameters are optional (no spaces should be included because they're not a character that is allowed to 
```

Comment:
If you really want to include the "no spaces can be included" information ("no spaces in a query" is something I would have said went without saying), it'd be better to reformat these two lines (lines 31 and 32) so all lines are of a more reasonable length.